src/views/Request.svelte

```diff
@@ -24,9 +25,9 @@ $: provider = request.provider || {}
 $: conversations = request.threads || []
 $: isMine = me.id && (requestor.id === me.id)
 $: imProviding = me.id && (provider.id === me.id)
-$: hasConversation = conversations.length > 0
+$: hasConversation = conversations.length > 0 || potentialConversation
 $: destination = request.destination && request.destination.description || ''
-$: if (!conversationId && conversations.length > 0) {
+$: if (!potentialConversation && !conversationId && conversations.length > 0) {
```

Comment:
assigning a `default` more declaratively would be helpful.

Reply:
I couldn't come up with a clear declarative way to do this, so for now I added a comment to at least provide a helpful mental clue as to what's happening. A better solution can be added later.

Reply:
understood